docs/assets/funfacts.txt

```diff
@@ -16,6 +16,21 @@ SkyClient was once exclusive to Windows users.\nFortunately, now we have a Java-
 You'll never believe this, but our Discord's support channels\nare in a category called SUPPORT IS HERE! Use them as necessary.
 SBE has skidded from countless mods including DSM, Skytils and NEU!
 SkyClient contains only the finest ethically sourced mods and packs -\nwe asked permission before adding them to SkyClient.
+In Canada, Santa's postal code is H0H 0H0.
+Zetvue makes music! You can too!
+Guvf vf abg na vzcbegnag fragrapr.
+ROT13
+Vs lbh qrpvcure guvf, lbh fubhyq yvfgra gb Mrgihr'f zhfvp.
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA\nAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+KTibow holds the record for the fastest promotion to admin.
+Play The Binding of Isaac instead of Hypixel Skyblock
+Play Terraria instead of Hypixel Skyblock
+Play Minecraft Survival instead of Hypixel Skyblock
+Play literally anything else instead of Hypixel Skyblock
+Just 30 minutes of daily complaining can physically damage your brain.\nListening to someone constantly complain also negatively affects brain function.
+The # symbol isn’t officially called a hashtag or a pound.\nIt’s technically called an "octothorpe." The “octo-” means “eight” to refer to its points.
+Try to do something creative! It could be music, art, coding...\nYou won't know how much you like it until you try it, so give it a shot :)
+Minecraft 1.8.9 is almost 7 years old.\nWhy are you playing severely outdated versions again?
```

Comment:
maybe just replace this with https://howoldisminecraft189.today/

Reply:
no

Reply:
the way this is phrased sounds weird
having a link to that site is more neutral but also could be interpreted in more ways

Reply:
eh nah